components/PeerReviews/PeerReviewPerson.tsx

```diff
@@ -0,0 +1,103 @@
+import AuthorAvatar from "~/components/AuthorAvatar";
+import { AuthorProfile } from "~/config/types/root_types";
+import { css, StyleSheet } from "aphrodite";
+import { ReactElement } from "react";
+import icons from "~/config/themes/icons";
+import colors from "~/config/themes/colors";
+import ReactTooltip from "react-tooltip";
+import { ID } from "~/config/types/root_types";
+
+type Props = {
+  id: ID,
+  status?: string,
+  authorProfile?: AuthorProfile,
+};
+
+function PeerReviewPerson({
+  id,
+  status,
+  authorProfile,
+}: Props): ReactElement {
+  
+  const getTooltipMessage = () => {
```

Comment:
it's usually not a good practice to create a functional component within a component for memoization purposes & how react elements work with life-cycle methods. It's better to have it as a const variable unless you're mutating values based on given args. 
```suggestion
  const tooltipMessage = (
```

Reply:
I like the suggestion for simplicity. Just for curiosity, can you share more details on:

> how react elements work with life-cycle methods

I'm curious as to the difference in performance between the two

Reply:
ah never mind, I saw below whatcha mean. Makes sense